MEDO v0.5.owl

```diff
@@ -804,6 +975,536 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
+        <skos1:related>MedBiquitous - AM13: Peer Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_summative_portfolio-based_assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_summative_portfolio-based_assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000105"/>
+        <obo:IAO_0000115>A type of formal summative assessment review of a learner&apos;s achievement of agreed-upon academic objectives or completion of a negotiated set of learning activities compiled in a learner&apos;s portfolio.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM14: Review of a learner&apos;s achievement of agreed-upon academic objectives or completion of a negotiated set of learning activities, based on a learner portfolio (Institute for International Medical Education, 2002) (&quot;a systematic collection of a student&apos;s work samples, records of observation, test results, etc., over a period of time&quot;— ERIC, 1994)</obo:IAO_0000119>
+        <rdfs:label>formal summative portfolio-based assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM14: Portfolio-Based Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_summative_research_or_project_assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_summative_research_or_project_assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000105"/>
+        <obo:IAO_0000115>A type of formal summative assessment of activities and outcomes (e.g., posters, presentations, reports, etc.) of a project in which the learner participated or conducted research.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM16: Assessment of activities and outcomes (e.g., posters, presentations, reports, etc.) of a project in which the learner participated or conducted research (Dyrbye, Davidson, &amp; Cook, 2008)
+
+Synonymous with/Includes: Multi-Media Production</obo:IAO_0000119>
+        <rdfs:label>formal summative research or project assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM16: Research or Project Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#formal_summative_self-assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#formal_summative_self-assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo//MEDO_0000105"/>
+        <obo:IAO_0000115>A type of formal summative assessment where student observes, evaluates, and reflects upon one’s own deficiencies, achievements, behavior or professional performance, and competencies. The assessment is completed by the learner to reflect upon and critically assess their own performance against a set of established criteria.</obo:IAO_0000115>
```

Comment:
My revision: "A type of formal summative assessment in which a student observes, evaluates, and reflects upon the student's own deficiencies, achievements, behavior or professional performance, and competencies. The assessment is completed by the student to reflect upon and critically assess the student's own performance against a set of established criteria.